app/bluetrax.py

```diff
@@ -0,0 +1,137 @@
+import asyncio
+import json
+from typing import List
+from datetime import datetime, timezone, timedelta
+from gql import gql, Client
+from gql.transport.httpx import HTTPXAsyncTransport
+import httpx
+import pydantic
+
+class User(pydantic.BaseModel):
+    user_id: str
+    client_id: str
+    contact_id: str
+    client_name: str
+
+class AuthenticateResult(pydantic.BaseModel):
+    users: List[User] = pydantic.Field(default_factory=list, alias="selectUsersByUsernamePassword")
+
+
+class UserAsset(pydantic.BaseModel):
+    unit_id: str
+    user_id: str
+    asset_id: str
+    currentLoc: dict
```

Comment:
minor: we could use an alias so the field name can be snake_case to be PEP8 compliant
https://docs.pydantic.dev/1.10/usage/model_config/#alias-precedence